src/app/loans/loans-view/loan-account-actions/loan-account-actions.component.html

```diff
@@ -6,5 +6,6 @@
     <mifosx-foreclosure *ngIf="actions.foreclosure"></mifosx-foreclosure>
     <mifosx-prepay-loan [dataObject]="actionButtonData" *ngIf="actions.prepay_loan"></mifosx-prepay-loan>
     <mifosx-make-repayment [dataObject]="actionButtonData" *ngIf="actions.make_repayment"></mifosx-make-repayment>
+    <mifosx-waive-interest *ngIf="actions.waive_interest"></mifosx-waive-interest>
```

Comment:
Here you need to pass the `[dataobject]` and receive the `input` in the `waive-interest.component.ts` file. Please refer to the implementation of other action buttons.